src/java/org/apache/cassandra/io/sstable/format/SSTableWriter.java

```diff
@@ -104,11 +107,21 @@ protected SSTableWriter(Builder<?, ?> builder, LifecycleNewTracker lifecycleNewT
         this.mmappedRegionsCache = builder.getMmappedRegionsCache();
         this.lifecycleNewTracker = lifecycleNewTracker;
 
-        Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS);
+        List<Component> existingsComponents = components.stream().filter(c -> descriptor.fileFor(c).exists()).collect(Collectors.toList());
+        if (!existingsComponents.isEmpty())
```

Comment:
We could use `anyMatch` instead of making a collection during the normal path (if necessary, gathering for the error message).